frontend/src/Editor/Box.jsx

```diff
@@ -102,12 +106,24 @@ export const Box = function Box({
   }
 
   const ComponentToRender = AllComponents[component.component];
-  const resolvedProperties = resolveProperties(component, currentState);
-  const resolvedStyles = resolveStyles(component, currentState);
-
+  const resolvedProperties = resolveProperties(component, currentState, null, customResolvables);
+  const resolvedStyles = resolveStyles(component, currentState, null, customResolvables);
   resolvedStyles.visibility = resolvedStyles.visibility !== false ? true : false;
 
-  const exposedVariables = currentState?.components[component.name] ?? {};
+  let exposedVariables = {};
+
+  if (component.parent) {
```

Comment:
This condition never passes when I try to run. No component's parent is getting correctly set on ListView. Probably because of that, the radio button `exposedVariables` are not being calculated correct. Thereby disabling it from correctly working.

Reply:
Yes, this logic seems to be flawed. Working on it now.